index.html

```diff
@@ -0,0 +1,110 @@
+                    <li class="about"><a href="./about.html">About</a></li>
+                </ul>
+            </div>
+        </header>
+        <div id="intro" class="intro">
+            <h2 class="hello">"Hello! Sharing world"</h2>
+            <h1 id="title" class="title">INTERNATIONAL BOOK FAIR IN JESUS MARIA 2022 </h1>
+            <p class="description">You are invited to the International Book Fair in Lima (IBF)
+                that will take place on Proceres park in Jesus Maria</p>
+            <h3 class="date">22 July, 2022</h3>
+            <p class="location">Lima, Perú</p>
+        </div>
+    </div>
+    <div id="program" class="main-program">
+        <h2 class="subtitle">Main Program</h2>
+        <div class="linea"></div>
+        <div class="div-list">
+            <div class="item-list">
+                <img src="./images/list1.png" alt="conference" class="images">
+                <h3 class="img-title">Opinion</h3>
+                <p class="img-des">Listen to the Speakers experts of the art of writing</p>
+            </div>
+            <div class="item-list">
+                <img src="./images/list2.png" alt="conversation" class="images">
+                <h3 class="img-title">Conversation</h3>
+                <p class="img-des">Share your ideas and opinions with other people in the place</p>
+            </div>
+            <div class="item-list">
+                <img src="./images/list3.png" alt="buy" class="images">
+                <h3 class="img-title">Store</h3>
+                <p class="img-des">The most variable selection of books</p>
+            </div>
+        </div>
+        <a href="#" class="square">See whole program</a>
+        <div class="space"></div>
+    </div>
+    <div id="speakers" class="featured-speakers">
+        <h2 class="speakers-title">Featured Speakers</h2>
+        <div class="linea"></div>
+        
+    </div>
+    <div id="sponsor" class="div-sponsor">
+        <h2 class="sponsor-title">Sponsored by</h2>
+        <div class="linea"></div>
+        <ul class="sponsor">
+            <li class="sponsor-img"><img src="images/el-comercio.png" alt="el-comercio" class="sponsor-img"></li>
+            <li class="sponsor-img"><img src="images/cato2.png" alt="catolica" class="sponsor-img"></li>
+            <li class="sponsor-img"><img src="images/rpp.png" alt="rpp-noticias" class="sponsor-img"></li>
+            <li class="sponsor-img"><img src="images/tvperu.png" alt="tv-Perú" class="sponsor-img"></li>
+            <li class="sponsor-img"><img src="images/bbva2.png" alt="bbva" class="sponsor-img"></li>
+        </ul>
+    </div>
+    <footer class="site-footer">
+        <div class="footer">
+            <h2 class="slogan"><i class="fa-solid fa-cubes"></i> J.B. Desing</h2>
+            <p class="copyright"><i class="fa-solid fa-copyright"> project make by Josue Benavides some Rights Reserved</i></p>
+        </div>
+    </footer>
```

Comment:
- [ ] The partner section and footer height are much. Kindly check how is the original and change it.
- [ ] Please follow the layout: composition and space between elements.
- [ ] In the mobile view partner layout changing and it is two lines. Your design is still one line. Kindly change it.
![image](https://user-images.githubusercontent.com/38635158/176969821-acf3a44a-83d2-4ed7-9ade-34ff56fb4ab7.png)
- [ ] These sections are also on the about.html page so kindly change them on this page.
___
### ORIGINAL
![image](https://user-images.githubusercontent.com/38635158/176969850-4c042114-6f48-47b2-ae19-21bde455a06d.png)
____
### YOUR DESIGN MOBILE VIEW
![image](https://user-images.githubusercontent.com/38635158/176970044-9ef7bbdf-c462-47d6-97b7-55a26466313f.png)

____
### ORIGINAL MOBILE VIEW
![image](https://user-images.githubusercontent.com/38635158/176969996-8612d0cf-c43a-46b1-a255-0c7fd2c75349.png)